ivre/analyzer.py

```diff
@@ -579,9 +625,12 @@ def search_anomalies(self, outliers, percent_mode=False):
         instead of `outliers` anomalies
         Update self._anomalies attribute
         """
-        # 0.01 is faster and works well most of the time
-        # but 0.5 works with small host number.
-        nu_percent = outliers * 0.01 if percent_mode else 0.5
+        if percent_mode:
+            nu_percent = outliers * 0.01
+        else:
+            # 0.01 is faster but causes troubles with a small host number
+            # 0.5 works with a small host number but takes longer
+            nu_percent = 0.5 if len(self._list_id) < 500 else 0.01
```

Comment:
Would it make sense here to have, instead of just two different values, a value computed from `len(self._list_id)`?

Maybe something like `max(.01, -0.0005 * len(self._list_id) + 0.55)`, or `.5 * 10 ** (-float(len(self._list_id))/100)`?